hello_world/src/hello-world.cpp

```diff
@@ -1,3 +1,4 @@
 #include "hello-world.hpp"
+#include "cmake_public.cpp"
```

Comment:
In C/C++ you almost never want to include source files. This should be the header file.